.github/workflows/codeql.yml

```diff
@@ -0,0 +1,77 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
could you remove this file from the PR and make a new one for it please

Reply:
File has been removed.